module/attribute/src/Resources/config/services.yml

```
@@ -25,7 +25,7 @@ services:
 
     Ergonode\Attribute\Infrastructure\Persistence\Projector\:
         resource: '../../Infrastructure/Persistence/Projector/*'
-        tags: ['messenger.message_handler']
+        tags: [ 'ergonode.es.projector' ]
```

```suggestion
        tags: [ 'ergonode.event_sourcing.projector' ]
```
On the one hand, the name of the composer package is `es` but it's actually event sourcing - more adequate to the rest of the application